guides/openai-assistant-py/openai_assistant/actions.py

```diff
@@ -18,6 +18,20 @@ def save_code_to_file(sandbox: Sandbox, args: Dict[str, Any]) -> str:
     except Exception as e:
         return f"Error: {e}"
 
+def write_to_file(sandbox: Sandbox, args: Dict[str, Any]) -> str:
+    filename = args["filename"]
+    content = args["content"]
+
+    try:
+        dir = os.path.dirname(filename)
+
+        sandbox.filesystem.make_dir(dir)
```

Comment:
This isn't bad, but not sure if it would be better to create function to create the dir separately

Reply:
Thanks, smart! Added new function and action, I think that way is easier for me to replicate in JS/TS version.